suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -37,16 +41,35 @@ export const discoveryConfigSlice = createSlice({
         toggleAreTestnetsEnabled: state => {
             state.areTestnetsEnabled = !state.areTestnetsEnabled;
         },
-        setDiscoveryStartTimestamp: (state, { payload }) => {
+        setDiscoveryStartTimestamp: (state, { payload }: PayloadAction<number | null>) => {
             state.discoveryStartTimeStamp = payload;
         },
+        setDisabledDiscoveryNetworkSymbolsForDevelopment: (
```

Comment:
Maybe it will be better to use something like `toggleDisabledDiscoveryNetworkSymbolForDevelopment` because it is doing both add/remove.

wdyt?

Reply:
https://github.com/trezor/trezor-suite/pull/10712/commits/71fb1e0c431b74b816cd1931fe1be27f0ec5dc21